storm.go

```diff
@@ -152,12 +153,26 @@ func toBytes(key interface{}, codec codec.MarshalUnmarshaler) ([]byte, error) {
 	if key == nil {
 		return nil, nil
 	}
-	if k, ok := key.([]byte); ok {
-		return k, nil
-	}
-	if k, ok := key.(string); ok {
-		return []byte(k), nil
+	switch t := key.(type) {
+	case []byte:
+		return t, nil
+	case string:
+		return []byte(t), nil
+	case int:
+		return itob(int(t)), nil
+	case int64:
+		return itob(int(t)), nil
+	case uint:
+		return itob(int(t)), nil
+	case uint64:
+		return itob(int(t)), nil
+	default:
+		return codec.Marshal(key)
 	}
+}
 
-	return codec.Marshal(key)
+func itob(v int) []byte {
+	b := make([]byte, 8)
+	binary.BigEndian.PutUint64(b, uint64(v))
```

Comment:
Mmmmm maybe we should use [`PutVarint`](https://golang.org/pkg/encoding/binary/#PutVarint) instead. It would allow people to use negative IDs if they want